src/plugins/dam/sagas/uploadFileFlow.js

```diff
@@ -30,9 +30,9 @@ export default function* (hashName, fileInfo, fileBuffer, preSignedUrl, variant)
 
   for (let retries = 0; retries < MAX_RETRIES; retries += 1) {
     try {
-      // Delay the process by 2 seconds if this isn't the first try
+      // Delay the process by 4 seconds if this isn't the first try
       if (retries) {
-        yield delay(2000);
+        yield delay(4000);
```

Comment:
@jalopez13 are you certain this is the fix?  this is for the upload process, not hte preview part correct?  i thought the broken thumb was the previewing of the asset and the upload was in fact still working.

seems like we may want to do the `yield call(checkImage, previewUrl)` multiple times before assuming a re-upload is needed.  also, that yield checkImage is currently wrong, it should be yield call(fun, args).

@rsumilang @efjacobson can you also check this out.

Reply:
@gdbrown 

I updated the delay back to 2 secs. It really makes no difference on the rendering of the thumbnail. Issue was that the component would render but the previewUrl was still not available hence the 403 error I was seeing. So I updated the component to render the thumbnail only when the previewUrl was ready. I tested this over and over and I always see a rendered thumbnail. Do you want me to make any other updates?

Reply:
Reverted to 2000!